### PR TITLE
Change logging.error to logging.excetion

### LINKED ISSUE #151: Log system is useless
The information being logged by the app is useless. It does not tell what was the error nor where it happened. Development and debugging process is terrible to do

### PR DESCRIPTION
**Pull request by ffdarkpenguin**
closes #151

### CODE REVIEW

File: oneview_redfish_toolkit/app.py [hunk 1]
```diff
@@ -186,17 +186,17 @@ def not_implemented(error):
     try:
         port = int(config["redfish"]["redfish_port"])
     except Exception:
-        logging.error("Port must be an integer number between 1 and 65536")
+        logging.exception("Port must be an integer number between 1 and 65536")
         exit(1)
     # Checking port range
     if port < 1 or port > 65536:
-        logging.error("Port must be an integer number between 1 and 65536")
+        logging.exception("Port must be an integer number between 1 and 65536")
```
Comment (ricardoas) [inline oneview_redfish_toolkit/app.py]: Here!

File: oneview_redfish_toolkit/app.py [hunk 2]
```diff
@@ -186,17 +186,17 @@ def not_implemented(error):
...
         exit(1)
 
     ssl_type = config["ssl"]["SSLType"]
     # Check SSLType:
     if ssl_type not in ('disabled', 'adhoc', 'certs', 'self-signed'):
-        logging.error(
+        logging.exception(
```
Comment (ricardoas) [inline oneview_redfish_toolkit/app.py]: Here!

File: oneview_redfish_toolkit/app.py [hunk 3]
```diff
@@ -221,7 +221,7 @@ def not_implemented(error):
                 logging.warning("Using existing self-signed certs")
 
         if ssl_cert_file == "" or ssl_key_file == "":
-            logging.error(
+            logging.exception(
```
Comment (ricardoas) [inline oneview_redfish_toolkit/app.py]: Here!

File: oneview_redfish_toolkit/util.py [hunk 4]
```diff
@@ -325,7 +325,7 @@ def generate_certificate(dir_name, file_name, key_length, key_type="rsa"):
         private_key.generate_key(OpenSSL.crypto.TYPE_DSA, key_length)
     else:
         message = "Invalid key_type"
-        logging.error(message)
+        logging.exception(message)
```
Comment (ricardoas) [inline oneview_redfish_toolkit/util.py]: Here!

File: oneview_redfish_toolkit/blueprints/network_adapter.py [hunk 5]
```diff
@@ -65,7 +65,7 @@ def get_network_adapter(uuid, device_id):
 
         if (device_id_validation - 1) < 0 or (device_id_validation - 1) >= \
             len(server_hardware["portMap"]["deviceSlots"]):
-            logging.error(server_hardware)
+            logging.exception(server_hardware)
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/network_adapter.py]: Here!

Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/blueprints/network_adapter.py]: remove, this was inserted for debugging and forgot to remove

File: oneview_redfish_toolkit/app.py (hunk 1 above)
Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/app.py]: fixed

File: oneview_redfish_toolkit/app.py (hunk 2 above)
Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/app.py]: fixed

File: oneview_redfish_toolkit/app.py (hunk 3 above)
Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/app.py]: fixed

File: oneview_redfish_toolkit/util.py (hunk 4 above)
Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/util.py]: fixed

**Pull request merged**